pkg/reporeader/readers/localfsreader.go

```diff
@@ -13,6 +13,16 @@ import (
 type LocalFSReader struct {
 }
 
+// GetRepoName implements reporeader.RepoReader.
+func (*LocalFSReader) GetRepoName() (string,error) {
+	wd, err := os.Getwd()
```

Comment:
💡 repo vs workingdir- the runt name is getRepo but actually all this is getting is working director, might be add more relevance and underlying assumptions of this function please? What do you think? Thanks

Reply:
yes a better comment there would help. the idea is to extract the name of the repo in order to be able to auto-fill the name of the deployment to present some kind of reasonable default. It would be cool to eventually extract that information using a vcs capability but most of the time the directory name is the repo name anyway so i was using that for now. The reason it is a separate function is to maintain the abstraction, which could include reading a remote repo from vcs in which case reading a directory name for the root wouldn't work